src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const lines = sourceString
+    .split(';')
+    .map((line) => line.trim())
+    .filter((line) => line.length > 0);
+
+  lines.forEach((line) => {
+    const [property, value] = line.split(':').map((part) => part.trim());
+
+    if (property && value !== undefined) {
```

Comment:
The condition `value !== undefined` is used to check if a value exists. However, this condition will also pass for empty strings. If empty values should be excluded, consider using `value.length > 0` instead.